app/views/admin/loan_questions/_loan_types.slim

```diff
@@ -0,0 +1,35 @@
+.required-loan-types
+  label = t('loan_questions.required_loan_types')
```

Comment:
I've always used single quotes unless interpolation is needed. You've never said anything about it before. Our style guide references bbatsov's style guide, which doesn't specify (but uses single quote style itself):
https://github.com/bbatsov/ruby-style-guide#strings
Personally, I don't have a problem with inconsistency in quoting style. If you do, we can talk about it, but there's currently nothing in the dev procedures doc about it.